### ISSUE TITLE
Search does not work

### ISSUE THREAD

**Issue by hhorak** [developer-portal/website#76]
Trying to search content that is for sure on the portal (KW: mariadb, mysql, postgresql), I get no results:
https://developer.fedoraproject.org/search.html?q=mysql

**Comment by hhorak** [comment]
Interestingly, searching works in the field in the header (on homepage), but the problem with results there is that both text and background have pretty similar color (blue), so the text is not readable.

**Comment by pvalena** [comment]
Yes, that'll be caused by using the no longer supported (and outdated) search plugin.

We have to replace/fix that.

**Comment by jackorp** [comment]
This issue was probably fixed.

**Comment by pvalena** [comment]
How's that? It was just hidden, search was removed, but it doesn't work - it wasn't replaced with anything...

**Comment by jackorp** [comment]
I was looking at the age of the issue and the fact that there was merged #82 . I have created #144 with more on-point and current info. Whether *this* particular issue was fixed is not clear to me, but it seems like it.

**Comment by pvalena** [comment]
Oh right, I've missed the other issue, which is replacing it :). To clarify, I consider it the same issue = there's no working search :) .. would close it as duplicate >).
